README.md

```diff
@@ -21,6 +21,8 @@ implicits and tools for:
 
 | play version | scala versions          | scalatest version | artifact name          | bintray                                                                                                                                                                                     |
 |--------------|-------------------------|-------------------|------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 2.8.x        | 2.13.1, 2.12.6, 2.11.12 |                   | play28-json-ops-v4     | [ ![Download](https://api.bintray.com/packages/rallyhealth/maven/play28-json-ops-v4/images/download.svg) ](https://bintray.com/rallyhealth/maven/play28-json-ops-v4/_latestVersion)         |
```

Comment:
These images don't work, are they supposed to before merging?

Reply:
AFAICT they get generated as part of the release process.